LucidGloves/src/ControllerPose.cpp

```diff
@@ -32,10 +35,13 @@ vr::DriverPose_t ControllerPose::UpdatePose() {
 			newPose.vecPosition[1] = trackedDevicePoses[m_shadowControllerId].mDeviceToAbsoluteTracking.m[1][3] + vectorOffset.v[1];
 			newPose.vecPosition[2] = trackedDevicePoses[m_shadowControllerId].mDeviceToAbsoluteTracking.m[2][3] + vectorOffset.v[2]; //- forward
 
-			vr::HmdQuaternion_t offset_quaternion = QuaternionFromAngle(1, 0, 0, DegToRad(-45));
-
+			//vr::HmdQuaternion_t offset_quaternion = QuaternionFromAngle(1, 0, 0, DegToRad(-45));
+			bool isRightHand = m_shadowDeviceOfRole == vr::TrackedControllerRole_RightHand;
```

Comment:
This won't be changing - perhaps best to add as a member variable?

Reply:
removed